src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -283,11 +284,14 @@ public static string TitleThe(string title)
             return title.Trim();
         }
 
-        public static string CleanFileName(string name, bool replace = true)
+        public static string CleanFileName(string name, NamingConfig namingConfig)

```

Comment:
Why not just pull the config using _namingConfigService.getConfig in this method instead of passing it in and having to add the dependence to every other file for namingConfigService? This is how the other functions in file name builder are setup. Seems cleaner.